### PR TITLE
MEPTS-765 | Changes on CompositionStrings for MQ Cat 7

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java [hunk 1]
```diff
@@ -96,6 +96,7 @@ public ReportDefinition constructReportDefinition() {
         new Parameter("startDate", "Data Inicial Inclusão", Date.class),
         new Parameter("endDate", "Data Final Inclusão", Date.class),
         new Parameter("revisionEndDate", "Data Final Revisão", Date.class),
-        new Parameter("location", "Unidade Sanitária", Location.class));
+        new Parameter("location", "Unidade Sanitária", Location.class),
+        new Parameter("testStart", "Testar Iniciar", Boolean.class));
```
Comment (Edgencio) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java]: Why do we need the parameter **testStart** ?? If it is useless the it should be removed

**Review by Edgencio — APPROVED** [review]
The ticket looks fine for me

File: api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java [hunk 2]
```diff
@@ -100,24 +100,6 @@ public void getMQ7patientsShouldPass() throws EvaluationException {
     assertNotNull(evaluatedCohort.getMemberIds());
   }
```
Comment (cfaife) [inline api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java:101]: why did you remove the test methods?

File: api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java [hunk 3]
```diff
@@ -136,24 +118,6 @@ public void getMQ7patientsShouldPass3() throws EvaluationException {
...
```
Comment (cfaife) [inline api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java:120]: why did you remove the test methods?

File: api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java (hunk 2 above)
Comment (helderjosue) [reply api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java:101]: Edgencio said  that I should remove the methods instead of commenting so the code could be built by the others

Comment (cfaife) [reply api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/QualityImprovement2020CohortQueriesTest.java:101]: but these methods were already there  before your implementation

**Review by cfaife — APPROVED** [review]
